Source/WebCore/platform/graphics/skia/PlatformDisplaySkia.cpp

```diff
@@ -128,6 +162,7 @@ class SkiaGLContext : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Ski
     std::unique_ptr<GLContext> m_skiaGLContext WTF_GUARDED_BY_LOCK(m_lock);
     sk_sp<GrDirectContext> m_skiaGrContext WTF_GUARDED_BY_LOCK(m_lock);
     mutable Lock m_lock;
+    int m_sampleCount { 0 };
```

Comment:
IMHO we should use unsigned consistently - I am aware that Skia uses int for the sampleCount parameter, but negative sample counts don't make any sense. So I think we should choose an unsigned.